deposit/src/main/java/edu/unc/lib/deposit/fcrepo4/IngestContentObjectsJob.java

```diff
@@ -284,6 +288,7 @@ private void ingestFileObjectAsWork(ContentContainerObject parent, Resource pare
 			WorkObject newWork = repository.createWorkObject(workPid, workModel);
 
 			// TODO add the FileObject's description to the work instead
```

Comment:
Right, I was just leaving those in untilI was finished, so I could easily find the spots I was workingon